en/06_Project_Governance/05_Major_release_policy.md

```diff
@@ -0,0 +1,135 @@
+
+Once a new major enters the beta period, it should be compatible with the dependencies intended to ship in the stable release. Substantial breaking changes should be avoided post-beta.
+
+Prior to the beta period, alpha releases are published at the discretion of the development team.
+
+### Stable release
+
+New major releases of Silverstripe CMS are tagged between April and June of odd years.
+
+### Active development
+
+Once a major release is stable, it enters a period of active development. A major release in active development:
+- Will receive regular minor releases that ships new feature and API in a backward compatible way.
+- Will receive regular patches for low impact bugs and low severity security issues.
+
+A major releases line will stay in "active development" of approximatively two years. "Active development" for a major release line ends when the follow up major release is tagged stable.
+
+Only one major release line is in "active development" at once.
+
+### Bug and security fixes
+
+A Silverstripe CMS major release line enters the "bug and security fixes" phase once the follow major release is tagged stable.
+
+Release lines in the "bug and security fixes" phase will receive:
+- patches for low impact bugs
+- patches for low low severity security issues. 
+
+Release lines in the "bug and security fixes" phase will not receive:
+- new feature or new APIs
+- new minor releases.
+
+A major release line stays in the "bug and security fixes" phase for 1 year.
+### Security fixes only
+
+At the end of "Bug and security fixes" phase, a Silverstripe CMS major release line transition to the "security fixes only" phase.
+
+In the "security fixes only" phase, a Silverstripe CMS major release line will only receive patches for high impact security issues defined has having a CVSS of 7.0 or above.
+
+### End-of-life
+
+A Silverstripe CMS major release line will be end-of-life once its follow up major release line exit active development.
+
+End-of-life major release do not receive any security fixes or bug fixes. 
+
+## PHP Support commitments
+
+Silverstripe CMS major releases tracks PHP releases. 
+
+The Silverstripe CMS release cycle is built around the assumptions that:
+- New PHP releases are shipped at the end of November on an annual basis.
+- PHP releases are in full support for two years with 1 year of limited support. 
+
+At launch, a Silverstripe CMS major release support all PHP version in full support. PHP versions in limited support are not supported at launch by new Silverstripe CMS majors.
+
+Following the initial release of a Silverstripe CMS major, we will aim to add forward compatibility for the next PHP release. e.g.: Silverstripe CMS 5 at launch will support PHP 8.1 and PHP 8.2. Will aim to add support for an eventual PHP 8.3 release to CMS 5.
+
+We try our best not to drop support for older PHP releases within Silverstripe CMS major release line even if those PHP release reach end-of-life.
```

Comment:
I would even go further and align this with the handling of fixed dependencies, because really, that's what PHP is, no?

```suggestion
Within a Silverstripe major release line, we will not drop support for the latest actively maintained PHP release at the time of the initial major Silverstripe release (e.g. PHP 8.2 for Silverstripe 5), even if that PHP release reaches end-of-life. The only exception is if the upgrade is necessary to fix a high impact bug or security vulnerability. 
```

When PHP 8.2 reaches end-of-life in 2025, Silverstripe 5 will only receive bug end security fixes anyway.

Reply:
I'm trying to find a balance between readability, clarity and not painting ourselves in a corner. 

I'll go with:
> Support for end-of-life PHP releases is not dropped within a Silverstripe CMS major release line, unless it's necessary to address vulnerabilities or high impact bugs.